test/test_differential_expression.py

```diff
@@ -95,9 +102,9 @@ def test_get_en_stats(conn):
     query.add_filter(GenomeQuery.Filter.REGULATION, GenomeQuery.Regulation.UP)
 
     result = en.get_differential_expression_stats({EN_TUTORIAL_FILE: query})
-    entries = result.values()[0]
+    entries = list(result.values())[0]
```

Comment:
I'm not sure how big this list will be. Maybe we should avoid unnecessary work here?
```suggestion
    entries = next(iter(result.values()))
```

Reply:
I guarantee you that the list is exactly 1 item long, but `next(iter(...))` looks better and cooler indeed (`list()` has been added automatically).